src/demo/App.js

```diff
@@ -1,7 +1,8 @@
 /* eslint no-magic-numbers: 0 */
 import React, { useState }  from 'react';
 
-import { Dropdown, CheckboxList, ComponentTemplate, ExpandableMenuItem, Accordion } from '../lib';
+import { Dropdown, CheckboxList, ComponentTemplate, ExpandableMenuItem, Accordion, Tab2 } from '../lib';

```

Comment:
Is Tab2 still needed?
```suggestion
import { Dropdown, CheckboxList, ComponentTemplate, ExpandableMenuItem, Accordion } from '../lib';

```